app/__init__.py

```diff
@@ -115,8 +115,9 @@ def create_app():
 
     # development api
     with app.app_context():
+        db.create_all()
```

Comment:
Remove

Reply:
@iamareebjamal I didn't find any other way to solve the problem. It would be great if you tell me how to solve this.

Reply:
Refactor your code in such a way that it does not query the database at the time of application bootstrap process